Source/BluetoothError.swift

```diff
@@ -78,6 +79,8 @@ extension BluetoothError: CustomStringConvertible {
             return "Connection error has occured: \(err?.localizedDescription ?? "-")"
         case let .peripheralDisconnected(_, err):
             return "Connection error has occured: \(err?.localizedDescription ?? "-")"
+        case let .peripheralDisconnectionFailed(_, err):
+            return "Connection error has occured: \(err?.localizedDescription ?? "-")"
```

Comment:
According to documentation disconnection cannot fail. So there shouldn't be any error related to that.

Reply:
+1